### PR TITLE
docs: .nojekyll in static folder.

### PR DESCRIPTION
**Pull request by sutanmufti**
### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [x] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### Edits

Please ensure that 'Allow edits from maintainers' is checked. PRs without this option may be closed.

---

adding the following instruction: make sure `.nojekyll` exists without the github page config.

My note: the `.nojekyll` instruction is missing in [#10083](https://github.com/sveltejs/kit/pull/10083). People that do not use custom github pages config will be confused (like me). I read that before, and I noticed that the instruction is missing. Moreover, people debugging will read this very popular [thread](https://stackoverflow.com/questions/75497065/sveltekit-deployment-on-gh-pages-doesnt-work-only-html-is-shown-css-and-js) and get mislead because the instruction to add `.nojekyll` is missing.

### CODE REVIEW

File: documentation/docs/25-build-and-deploy/50-adapter-static.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ When building for [GitHub Pages](https://docs.github.com/en/pages/getting-starte
 
 You'll also want to generate a fallback `404.html` page to replace the default 404 page shown by GitHub Pages.
 
+
+
```
Comment (Rich-Harris) [inline documentation/docs/25-build-and-deploy/50-adapter-static.md:90]:
```suggestion
```

**Comment by Rich-Harris** [comment]
Can you elaborate on why this only applies if you're not using the suggested workflow? (I don't use GitHub Pages so I don't really understand its significance — would it be simpler to just recommend that people always add `.nojekyll`?)

**Comment by sutanmufti** [comment]
Hi @Rich-Harris, of course. 

When the build file is in a separate GitHub repository from the source Svelte code repository. For instance, [this repo (by me)](https://github.com/sutanmufti/hsr-indonesia-production), that shares the built code without sharing the source svelte code (private github repo). See the github page result [here](https://sutanmufti.github.io/hsr-indonesia-production/).

That [build repo](https://github.com/sutanmufti/hsr-indonesia-production) is a [Github submodule](https://gist.github.com/gitaarik/8735255) of my original [svelte repository](https://github.com/sutanmufti/indonesia-hsr) which obviously you can't see. The build repo is static `index.html` page, not a svelte page; no need to build on workflow.

**Comment by Rich-Harris** [comment]
Got it, thanks. I moved the comment beneath the code snippet, since it's weird to say 'use this workflow (if you're not using the workflow, do this), here's the workflow'

**Review by Rich-Harris — APPROVED** [review]

**Pull request merged**